data/phi0474/phi031/phi0474.phi031.perseus-lat1.xml

```diff
@@ -711,7 +737,7 @@ Added to repository. Passes SGML validation.
 							>d</foreign></hi>: extulisse <hi rend="italics">E</hi></note>, ut
 				praeclarum videlicet munus atque instrumentum tribunatus ad aliquem, si nactus
 				esses, qui tuo arbitrio tribunatum gereret, deferre posses
-						<note><lemma>posses</lemma>
+						<note><w type="lemma">posses</w>
 					<hi rend="italics">duo fragmenta a Quintil.</hi> ix. 2. 54 <hi rend="italics">et
```

Comment:
Things to be aware of:
notes are sometimes not corrected tagged (wrong language). I think these are ok as they appear to be app crit in Latin but most are not and many need language attributes

I think the rend="italic" is better than "italics". 

(Again, most `<hi>` tags are meaningless. We generally like to address these by giving them semantic meaning but understandable if not possible or preferable here.)